tdrs-backend/tdpservice/email/tasks.py

```diff
@@ -69,3 +73,84 @@ def email_admin_num_access_requests():
                                    subject,
                                    email_context,
                                    )
+
+
+@shared_task
+def send_data_submission_reminder(due_date, reporting_period, fiscal_quarter):
+    """Send all Data Analysts a reminder to submit if they have not already."""
+    def get_fiscal_year(calendar_year, fiscal_quarter):
```

Comment:
could you add this as a util function in https://github.com/raft-tech/TANF-app/blob/156d26b04b45aa9e4d8ee5b7c6333a4b4234b9a4/tdrs-backend/tdpservice/parsers/util.py#L123